src/models/PermissionMap.js

```diff
@@ -99,10 +99,43 @@
       /**
        * Handles string based redirection for rejected permissions
        */
-      function handleStringRedirect(state, toParams, options) {
-        $state.go(state, toParams, options);
+      function handleStringRedirect(sref, toParams, options) {
+        var parsedSref = parseStateRef(sref);
+
+        $state.go(parsedSref.state, parsedSref.param || toParams, options);
+      }
+
+      /**
+       * Parse a stateRef expression like in ui-sref directive. The state is mandatory but stateParameters are optional.
+       * @private
+       *
+       * @param sref
+       * @returns {{state: String, paramExpr: (Object|null)}}
+       */
+      function parseStateRef(sref) {
+        var parsed = sref.replace(/\n/g, ' ').match(/^([^(]+?)\s*(\((.*)\))?$/);
```

Comment:
I got it from ui-sref implementation in function parseStateRef:
https://github.com/angular-ui/ui-router/blob/dcbaebf105c8ac11716e2ff1af181194b461d650/src/ng1/stateDirectives.ts

I simplified a bit the parseStateRef method because they authorize something like `ui-sref="{param:value}"` to avoid writing `ui-sref="current-state({param:value})"`. I thought it was not usefull in our case.